test/tokenbridge.test.ts

```diff
@@ -202,341 +216,318 @@ describe("TokenBridge", async function () {
+  })
+
+  it('Deposit staticADai and staticAUsdc', async () => {
+    // approve L1 bridge with max uint256 amount
+    await l1StaticDai.connect(l1user).approve(l1TokenBridge.address, MAX_UINT256);
+    await l1StaticUsdc.connect(l1user).approve(l1TokenBridge.address, MAX_UINT256);
+
+    // l1user deposits 30 staticADai and 40 staticAUsdc on L1 for l2user on L2
+    l1InitialStaticADaiBalance = await l1StaticDai.balanceOf(l1user.address);
+    await l1TokenBridge.connect(l1user).deposit(l1StaticDai.address, BigInt(l2user.starknetContract.address), 30);
+    expect(await l1StaticDai.balanceOf(l1user.address)).to.equal(l1InitialStaticADaiBalance-30);
+
+    l1InitialStaticAUsdcBalance = await l1StaticUsdc.balanceOf(l1user.address);
+    await l1TokenBridge.connect(l1user).deposit(l1StaticUsdc.address, BigInt(l2user.starknetContract.address), 40);
+    expect(await l1StaticUsdc.balanceOf(l1user.address)).to.equal(l1InitialStaticAUsdcBalance-40);
+
+    expect(await l1StaticDai.balanceOf(l1TokenBridge.address)).to.equal(30);
+    expect(await l1StaticUsdc.balanceOf(l1TokenBridge.address)).to.equal(40);
+
+    // flush L1 messages to be consumed by L2
+    expect(await l2StaticADai.call('get_last_update', {})).to.deep.equal({ block_number: {high: 0n, low: 0n} });
+    expect(await l2StaticAUsdc.call('get_last_update', {})).to.deep.equal({ block_number: {high: 0n, low: 0n} });
+
+    expect(await l2StaticADai.call('balanceOf', { account: BigInt(l2user.starknetContract.address) })).to.deep.equal({ balance: { high: 0n, low:  0n } });
+    expect(await l2StaticAUsdc.call('balanceOf', { account: BigInt(l2user.starknetContract.address) })).to.deep.equal({ balance: { high: 0n, low:  0n } });
+
+    const flushL1Response = await starknet.devnet.flush();
+    const flushL1Messages = flushL1Response.consumed_messages.from_l1;
+    expect(flushL1Response.consumed_messages.from_l2).to.be.empty;
+    expect(flushL1Messages).to.have.a.lengthOf(4);
+    expectAddressEquality(flushL1Messages[0].args.from_address, l1TokenBridge.address);
+    expectAddressEquality(flushL1Messages[0].args.to_address, l2TokenBridge.address);
+    expectAddressEquality(flushL1Messages[0].address, mockStarknetMessagingAddress);
+    expectAddressEquality(flushL1Messages[1].args.from_address, l1TokenBridge.address);
+    expectAddressEquality(flushL1Messages[1].args.to_address, l2TokenBridge.address);
+    expectAddressEquality(flushL1Messages[1].address, mockStarknetMessagingAddress);
+
+    // check balance and last update of L2 tokens
+    expect(await l2StaticADai.call('balanceOf', { account: BigInt(l2user.starknetContract.address) })).to.deep.equal({ balance: { high: 0n, low: 30n } });
+    expect(await l2StaticAUsdc.call('balanceOf', { account: BigInt(l2user.starknetContract.address) })).to.deep.equal({ balance: { high: 0n, low:  40n } });
+    // TODO enable these when the updated 
```

Comment:
There seems to be a `get_last_update` in ETHStaticAToken, so these should be enabled?

Reply:
The timestamp doesn't update as expected because it depends on the index and block number pr

Reply:
This test also needs to be revisited with the coming updates to token bridge